tests/operators/test_great_expectations.py

```diff
@@ -10,55 +10,291 @@
     pytest -p no:warnings
 """
 
-from contextlib import contextmanager
 import logging
 import os
-from pathlib import Path
 import unittest.mock as mock
-
-from airflow.exceptions import AirflowException
-from great_expectations_provider.operators.great_expectations import (
-    GreatExpectationsOperator,
-)
-from great_expectations.exceptions.exceptions import GreatExpectationsError
+from contextlib import contextmanager
+from pathlib import Path
 
 import pytest
+from airflow.exceptions import AirflowException
+from great_expectations.core.batch import BatchRequest
+from great_expectations.data_context.types.base import (CheckpointConfig,
+                                                        DataContextConfig)
+from great_expectations.exceptions.exceptions import CheckpointNotFoundError
 
+from great_expectations_provider.operators.great_expectations import \
+    GreatExpectationsOperator
 
 logger = logging.getLogger(__name__)
 
 # Set relative paths for Great Expectations directory and sample data
 base_path = Path(__file__).parents[2]
-good_data_file = os.path.join(
-    base_path, "include", "data/yellow_tripdata_sample_2019-01.csv"
-)
-bad_data_file = os.path.join(
-    base_path, "include", "data/yellow_tripdata_sample_2019-02.csv"
-)
+data_dir = os.path.join(base_path, "include", "data")
+
 ge_root_dir = os.path.join(base_path, "include", "great_expectations")
 
+# Test
+# bigquery
+# GCP
+# postgres
+# pandas
+# astronomer
```

Comment:
What is this comment for?

Reply:
Ah forgotten testing list. Thanks for the ping!